candihelper/helper.go

```diff
@@ -119,10 +119,17 @@ func ParseToQueryParam(source interface{}) (s string) {
 			continue
 		}
 
-		switch field.Interface().(type) {
-		case string:
+		dataType := reflect.ValueOf(field.Interface()).Type().Kind()
+		switch dataType {
+		case reflect.String:
 			val := url.PathEscape(field.String())
 			uri = append(uri, fmt.Sprintf("%s=%s", key, val))
+		case reflect.Ptr:
+			val := ""
+			if !field.IsNil(){
+				val = fmt.Sprintf("%v", field.Elem())
+			}
+			uri = append(uri, fmt.Sprintf("%s=%s", key, val))
```

Comment:
sekalian tambahin escape kayak yg diatas

```suggestion
			uri = append(uri, fmt.Sprintf("%s=%s", key, url.PathEscape(val)))
```